### DISCUSSION TITLE
Welcome to Flow Discussions!

### DISCUSSION THREAD

**Discussion by fotescodev** [onflow/flow#1220] [General]
To drive more community engagement and a tighter feedback loop, the Flow DevEx team is giving Github Audiences a try 🫡

Feel free to list any questions, issues, or bugs tormenting your developer journey. Let's discuss, surface issues, and prioritize together. 

What is your main pain point when developing on Flow? What would you like to see improved?

Yours,

Flow DevEx Team

**Comment by MrDSGC** [comment]
👋
